components/TokenBalances/TokenBalances.js

```diff
@@ -89,7 +89,7 @@ const TokenBalances = ({ tokenBalances, tokenValues, header, singleCurrency, sho
 									</div>
 									<div className="text-lg ">{usdValue}</div>{' '}
 									<div className="text-lg ">
-										{formattedVolume}{'\xa0'}
+										{Math.round(formattedVolume*Math.pow(10, 10))/Math.pow(10, 10)}{'\xa0'}
```

Comment:
Rather than performing this conversion in the HTML, can we perform it above when we declare `formattedVolume`? Perhaps just using the `.toFixed(10)` method like we do for the value.

Just a cleaner separation of concerns. Good to keep all formatting operations near eachother.